cmd/ido2icingadb/README.md

```diff
@@ -0,0 +1,113 @@
+# IDO2IcingaDB
+
+![Icinga Logo]
+
+#### Table of Contents
+
+- [About](#about)
+- [License](#license)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Support](#support)
+- [Contributing](#contributing)
+
+## About
+
+This commandline tool migrates history data from [IDO] to [Icinga DB].
+Or, more precisely: from the IDO SQL database to the Icinga DB one.
+
+!!! info
+
+    Everything else is already populated by Icinga DB itself.
+    Only the past history data of existing IDO setups
+    isn't known to Icinga DB without migration from IDO.
+
+## License
+
+This tool and its documentation are licensed under the terms
+of the GNU General Public License Version 2,
+you will find a copy of this license in the [LICENSE] file.
+
+## Installation
+
+1. Add the official release repository as described
+   in the [installation chapter] of the Icinga DB documentation.
+2. Install the package `ido2icingadb`.
+
+## Usage
+
+### Icinga DB
+
+1. Make sure Icinga DB is up, running and writing to its database.
+2. Optionally disable Icinga 2's IDO feature.
```

Comment:
In general, should mention limitations of the migration tool and how this affects the period where both write history. Also should provide a little more detailed instructions that keep this period as small as possible.